src/app/patient-summary/types/extensions/contact.types.ts

```diff
@@ -1,5 +1,5 @@
 import { Address } from "./address.types";
-import { CodeableConcept } from "./codeableConcept.tpes";
+import { CodeableConcept } from "./codeableConcept.types";
```

Comment:
Just double check if this file still exists? I think this has been removed instead of the typo corrected in the file name

Reply:
Yes, It was indeed removed, I'll revert the file, Thanks !